requirements.txt

```diff
@@ -1,5 +1,6 @@
 flake8
 mock
+pluggy
```

Comment:
Is ‘pluggy’ used anywhere?

Reply:
@ilazakis good question. Travis was complaining that it required a newer version of pluggy. `tox` was already installing pluggy but an older version. Smth changed in Travis and the build was happy with the addition of this requirement.